Microsoft.Toolkit.Uwp.SampleApp/SamplePages/GazeInteraction/GazeInteractionPage.xaml.cs

```diff
@@ -50,6 +53,13 @@ public void OnXamlRendered(FrameworkElement control)
                     gazeButtonControl.StateChanged += GazeButtonControl_StateChanged;
                 }
             }
+
+            gazePointer = GazeInput.GetGazePointer(control.FindParent<Page>());
```

Comment:
Look at how GetGazePointer is implemented! GetGazePointer(null) might be better, or GetGazePointer(this).